src/scenarios.py

```diff
@@ -0,0 +1,148 @@
+import argparse
+import csv
+import yaml
+
+import numpy as np
+from itertools import product
+
+from sim import simulate as sim
+
+#TODO add charger_density once supported in sim.py
+#TODO include support for depot_chargers. Not sure the best way to scale this
+# to the number of actual chargers that would be present at a depot.
+def genScenarios(battery_cap_kwh,nondepot_charger_rate,):
+    '''Generates a list of tuples of scenarios
+        args: all tuples of a start (inclusive), end (exclusive), and step 
```

Comment:
But this makes the nargs=3 thing make more sense. Maybe we should roll back to that.

Reply:
But does packing all of this into a CLI make sense versus running the analysis out of a fixed script?

Reply:
which bit, the inputs or outputs, or both?

Reply:
I mean, we should try to avoid running a final analysis off of the command line, unless we're interfacing with other languages and tools, since we then need to encapsulate all the analysis calls in a separate Bash script for reproducibility. If the analysis is being powered by command-line calls, it's probably better to run it out of a Python script.